### DISCUSSION TITLE
Should we rename `slurm-bundles` to `charmed-hpc-tofu`?

### DISCUSSION THREAD

**Discussion by NucciTheBoss** [canonical/hpc-team#9] [General]
As the title suggests, I'm wondering if we should rename the `slurm-bundles` repository to `charmed-hpc-tofu` :thinking: 

Reason saying is that as we're putting together some Tofu (OpenTofu Terraform) plans, I'm seeing that we won't just deploying Slurm, but a slew of other components such as COS, the file system, software stack infrastructure, etc, so I'm wondering if the name `slurm-bundles` would be a bit of a misnomer? It might lead people to believe that they are only deploying Slurm, and not that they're getting the OpenTofu modules necessary to deploy a functional Charmed HPC cluster and not just the workload scheduler.

Thoughts @jamesbeedy @wolsen @jedel1043?

**Comment by wolsen** [comment]
I think we should just have a different repository and not repurpose the slurm-bundles repo. You can still use juju bundles, but won't be able to register new ones. Bundles are still valid, especially for current juju versions and for LTS versions.

**Reply by NucciTheBoss** [reply]
Different repository makes sense if we want to keep the traditional Slurm bundles around. I was coming at it from the perspective that the Terraform/Tofu plans would be our "preferred" way of deploying Charmed HPC/Slurm, so we wouldn't want folks to stumble upon the Slurm bundles and use that instead.

However, it does make sense to keep the Slurm bundles if we still want to maintain support for Juju bundles. We'll likely want to make some "quality of life" updates however as Slurm bundles provides overlays for integrations that the Slurm charms do not have anymore.

**Reply by jedel1043** [reply]
On the flipside, if we want to be tool-agnostic with our naming (Terraform vs OpenTofu), I think `slurm-bundles`/`charmed-hpc-bundles` is a perfectly fine name for that, and I can see that it could be useful for us to centralize all our deployment suites into a single place, instead of having multiple places where users can stumble upon for deployment solutions.

**Reply by NucciTheBoss** [reply]
I see two things here:

1. Should the name for our new bundles be `charmed-hpc-bundles`?
2. Should we revamp the old `slurm-bundles` repository - either update the current bundles or replace with `charmed-hpc-bundles` - or should we preserve it as a public archive?

I think `charmed-hpc-bundles` should be the new repository name, and we should make the `slurm-bundles` repository a public archive so folks can still reference it after we've built up the Charmed HPC plans.

**Reply by jamesbeedy** [reply]
+1 @NucciTheBoss

**Comment by NucciTheBoss** [comment]
Okay, thank you all for chiming in!

I've set up the new bundles repository here: https://github.com/charmed-hpc/charmed-hpc-bundles.

Once we have it built up nicely with a README and some initial tf plans, we can be begin to work on tombstoning the `slurm-bundles` repository and preserve it as a point of reference.
